packages/epo-react-lib/src/atomic/Progress/Radial/ProgressRadial.tsx

```diff
@@ -0,0 +1,65 @@
+import { FunctionComponent } from "react";
+import { BaseProgressProps } from "@/types/progress";
+import * as Styled from "./styles";
+
+interface ProgressRadialProps extends BaseProgressProps {
+  darkMode?: boolean;
+}
+
+const ProgressRadial: FunctionComponent<ProgressRadialProps> = ({
+  min = 0,
+  max = 100,
+  value,
+  labelledById,
+  className,
+  markerFormatter,
+  darkMode = false,
+}) => {
+  const baseRadius = 50;
+  const baseWidth = 14;
+  const offsetRadius = baseRadius - baseWidth / 2;
+  const circumference = Math.PI * (offsetRadius * 2);
+
+  const isIndeterminate = value === undefined || isNaN(value);
+  const safeValue = Math.min(Math.max(value || 0, min), max);
+  /** allows min and max to be any value with the final rendered
+   *  value converted to a percentage to use in width or left offset.
+   */
+  const renderValue = ((safeValue - min) / (max - min)) * circumference;
+
+  return (
+    <Styled.Container
+      role="progressbar"
```

Comment:
This is my first time encountering the `progressbar` role and associated `aria` attrs.  I think, judging from [this write up](https://www.w3.org/WAI/ARIA/apg/patterns/meter/) (I also like [this write up of all of the "range" patterns](https://www.w3.org/WAI/ARIA/apg/practices/range-related-properties/)), `meter` may be a more applicable role for our use case.  And/or perhaps we default to either `progressbar` or `meter` but then include a prop to override the role so we can reuse the radial and bar versions across various implementations since the aria parts seem interchangeable regardless of role?

Reply:
updated to use a `role` prop that defaults to `meter`